lms/templates/wiki/history.html

```diff
@@ -228,9 +228,9 @@ <h2 id="preview-title">{% trans "Wiki Revision Preview" %}<span class="sr">, {%
       {% endif %}
       </div>
     </div>
-  </section>
+  </div>
 
-  <section id="mergeModal" class="modal" aria-hidden="true">
+  <div id="mergeModal" class="modal" aria-hidden="true">
```

Comment:
See my comment right above yours. :) The section tag was previously improperly closed with a div tag.